test/passing/ite_fit_or_vertical.ml.ref

```diff
@@ -0,0 +1,137 @@
+let _ = if b then e else ( e1 ; e2 )
```

Comment:
I can't help but feel there shouldn't be spaces inside the parentheses above.
It looks to me like the `indicate_multiline_delimiters` option is used incorrectly here. That is, if it's on ocamlformat will always print spaces, even though the parentheses might be on the same line.